src/flutter/flutter-wizzard.ts

```diff
@@ -0,0 +1,115 @@
+  const pubspecFile = findFile(projectDir, 'pubspec.yaml');
+
+  // ======== STEP 1. Add sentry_flutter and sentry_dart_plugin to pubspec.yaml ============
+  clack.log.step(
+    `Adding ${chalk.bold('Sentry')} to your apps ${chalk.cyan('pubspec.yaml',)} file.`,
+  );
+  const pubspecPatched = await traceStep('Patch pubspec.yaml', () =>
+    codetools.patchPubspec(
+      pubspecFile,
+      selectedProject.slug,
+      selectedProject.organization.slug
+    ),
+  );
+  if (!pubspecPatched) {
+    clack.log.warn(
+      "Could not add Sentry to your apps pubspec.yaml file. You'll have to add it manually.\nPlease follow the instructions at https://docs.sentry.io/platforms/flutter/#install",
+    );
+  }
+  Sentry.setTag('pubspec-patched', pubspecPatched);
+
+  // ======== STEP 2. Add sentry.properties with auth token ============
+
+  const propertiesAdded = traceStep('Add sentry.properties', () =>
+    codetools.addProperties(pubspecFile, authToken),
+  );
+  if (!propertiesAdded) {
+    clack.log.warn(
+      `We could not add "sentry.properties" file in your project directory in order to provide an auth token for Sentry CLI. You'll have to add it manually, or you can set the SENTRY_AUTH_TOKEN environment variable instead. See https://docs.sentry.io/cli/configuration/#auth-token for more information.`,
+    );
+  } else {
+    clack.log.info(
+      `We created "sentry.properties" file in your project directory in order to provide an auth token for Sentry CLI.\nIt was also added to your ".gitignore" file.\nAt your CI enviroment, you can set the SENTRY_AUTH_TOKEN environment variable instead. See https://docs.sentry.io/cli/configuration/#auth-token for more information.`,
+    );
+  }
+  Sentry.setTag('sentry-properties-added', pubspecPatched);
+
+  // ======== STEP 3. Patch main.dart with setup and a test error snippet ============
+  clack.log.step(
+    `Patching ${chalk.bold('main.dart')} with setup and test error snippet.`,
+  );
+
+  const mainFile = findFile(projectDir, 'main.dart');
+  const dsn = selectedProject.keys[0].dsn.public;
+
+  const mainPatched = traceStep('Patch main.dart', () =>
+    codetools.patchMain(mainFile, dsn),
+  );
+  if (!mainPatched) {
+    clack.log.warn(
+      "Could not patch main.dart file. You'll have to manually verify the setup.\nPlease follow the instructions at https://docs.sentry.io/platforms/flutter/#verify",
+    );
+  }
+  Sentry.setTag('main-patched', mainPatched);
+
+  // ======== OUTRO ========
+
+  const issuesPageLink = selfHosted
+    ? `${sentryUrl}organizations/${selectedProject.organization.slug}/issues/?project=${selectedProject.id}`
+    : `https://${selectedProject.organization.slug}.sentry.io/issues/?project=${selectedProject.id}`;
+
+  clack.outro(`
+    ${chalk.greenBright('Successfully installed the Sentry Flutter SDK!')}
+    
+    ${chalk.cyan(
+      `You can validate your setup by launching your application and checking Sentry issues page afterwards
```

Comment:
Should it make sense to give more specific instructions? Since we set up the plugin as well the app should be able to have symbolicated stacktraces. Telling the user to start the app I assume most people will just run it in dev mode

Reply:
btw @denrase wizard filename is wrong -> `wizzard`

Reply:
This message is the same as with the other wizards.